modules/installation-custom-aws-vpc.adoc

```diff
@@ -96,7 +96,7 @@ In case you need to create a public subnet in the AWS Outposts, verify that this
 To access your local cluster over your local network, the VPC must be associated with your Outpost's local gateway route table. For more information, see link:https://docs.aws.amazon.com/outposts/latest/userguide/outposts-local-gateways.html#vpc-associations[VPC associations] in the AWS Outposts User Guide.
 ====
 endif::aws-outposts[]
-* The VPC must not use the `kubernetes.io/cluster/.*: owned` tag.
+* The VPC must not use the `kubernetes.io/cluster/.*: owned`, `Name`, and `openshift.io/cluster` tags.
```

Comment:
@xenolinux Hi, I suggest explaining the reason why 'Name' cannot be used at the bottom of this line, similar to the description of 'kubernetes. io/cluster/. *: shared' tag in line 101 of the document.

Reply:
@shaozha I updated the PR- can you PTAL?